_opensearch/query-dsl/full-text.md

```diff
@@ -316,9 +339,9 @@ GET _search
 }
 ```
 
-## Simple query string
+### Simple query string
 
-The simple query string query is like the query string query, but it lets advanced users specify many arguments directly in the query string. The query discards any invalid portions of the query string.
+Use the `query_string_query` type to specify many arguments directly in the query string delineated by regular expressions. Searches with this type will discard any invalid portions of the string.
```

Comment:
Does this suggested change keep the meaning"
"Use the `query_string_query` type to specify directly in the query string multiple arguments delineated by regular expressions."